### DISCUSSION TITLE
Where is the install path on Windows?

### DISCUSSION THREAD

**Discussion by cdsedgwick** [nextfoam/baram#42] [Q&A] [answered]
I've run the installer on https://nextfoam.github.io/baram-pages/docs/installation/, which appears to complete without issue... but there's no feedback on where it was installed to, or how to actually run it. It also doesn't appear in Windows' list of applications. Searching the file system for "BARAM" yields no results... did the installation silently fail, or is this expected behaviour and I'm missing something about how to actually use it?

**Comment by cdsedgwick** [comment, marked as answer]
Looks like it the program installing into an admin directory when run as admin, instead of the AppData folder on C:\\ as expected. It didn't appear as the admin folder is restricted for other users.

**Comment by jiban** [comment]
Thank you for the update. It would help others with the same problem.

**Comment by jiban** [comment]
Hi, cdsedgwick.
I've just released v23.3.0 of BARAM, which comes with a new application called **_BaramMesh_**.
Have a look at the [screenshots](https://baramcfd.org/docs/screenshots/#barammesh).
